deploy.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# This script automates the following steps:
+# - Tag a version on the main branch
+# - Update the corresponding docker-compose.yml on the prod server
+# - Restart the corresponding docker container on the prod server
+
+# Exit if a command failed
+set -e
+
+packagesUrl="https://github.com/mint-o-badges/badgr-server/pkgs/container/badgr-server"
+productionUrl="openbadges.education"
```

Comment:
I'm not too familiar with bash scripts, but should `productionUrl`here be `api.openbadges.education`?